kvmagent/kvmagent/test/utils/snapshot_utils.py

```
@@ -1,5 +1,12 @@
 import os
 
+try:
+    import zstacklib
+except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)